Handler/Package.hs

```diff
@@ -6,6 +6,8 @@ module Handler.Package
     ( getPackageR
     , getPackageSnapshotsR
     , packagePage
+    , getPackageBadgeLtsMajorR
```

Comment:
Why not also support a `getPackageBadgeLtsR`, for "just tell me the latest version in any LTS"?